### PR TITLE
Adds PG16 support to TopN extension

### PR DESCRIPTION
**Pull request by naisila**
Note to reviewer: any idea on what to do with `customer_reviews_query` test file? I hate having 4 alternative outputs...
Update: changed topn command from 20 to 4 results - first 4 records don't change within different PG versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Takes the union of both `JSONB`s and returns a new `JSONB`.
 Sets the number of counters to be tracked in a `JSONB`. If at some point, the current number of counters exceed `topn.number_of_counters` * 3, the list is pruned. The default value is 1000 for `topn.number_of_counters`. When you increase this setting, `TopN` uses more space and provides more accurate estimates.
 
 # Compatibility
-`TopN` is compatible with the PostgreSQL 9.6, 10, 11, 12, and 13 releases. `TopN` is also compatible with all supported Citus releases, including Citus 6.x, 7.x, 8.x, and 9.x. If you need to run `TopN` on a different version of PostgreSQL or Citus, please open an issue. Opening a pull request (PR) is also highly appreciated.
+`TopN` is compatible with the PostgreSQL 9.6, 10, 11, 12, 13, 14, 15 and 16beta3 releases. `TopN` is also compatible with all supported Citus releases, including Citus 6.x, 7.x, 8.x, and 9.x. If you need to run `TopN` on a different version of PostgreSQL or Citus, please open an issue. Opening a pull request (PR) is also highly appreciated.
```
Comment (gurkanindibay) [inline README.md:147]: Do we need to write beta here? İf we don't need to change in PG16, we can write 16 instead of 16beta3

**Review by gurkanindibay — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hanefi) [reply README.md:147]: I like PG16 better.

Good catch by the way.

**Comment by hanefi** [comment]
The need for alternatife test outputs for test customer_reviews_query is quite bad. However I do not see an easy way to rewrite that query to generate same output on different PG versions.

I'm fine with adding a new output for the time being.

**Comment by naisila** [comment]
@hanefi I noticed that the first 4 rows are the same for all versions, so I will just change it from 20 to 4 results.

**Review by hanefi — APPROVED** [review]
Looks great now

**Review by gurkanindibay — APPROVED** [review]

**Pull request merged**